### PR TITLE
feat: retrieve languages from interaction definition and context

### PR DESCRIPTION
**Pull request by shaveko**
related to https://oat-sa.atlassian.net/browse/TR-3246

### Description 
PCI [configuration](https://www.imsglobal.org/spec/qti/v3p0/impl#h.1mc9puik2ft6) object is passed on pci instance creation. This PR brings population  of the config with language props.

### How to test

Using DevTools check pciConstructor.getInstance is called with configured language in config object

### CODE REVIEW

**Review by marpesia — APPROVED** [review]
Good job!!

**Review Checklist**
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

File: test/qtiCommonRenderer/interactions/pci/ims/test.js [hunk 1]
```diff
@@ -38,13 +47,83 @@ define(['taoQtiItem/qtiCommonRenderer/renderers/interactions/pci/ims'], function
...
+            language: 'ru_RU',
+            userLanguage: 'ru_RU'
+        });
 
         const instancePromise = ims().createInstance(interaction, context);
 
         assert.ok(instancePromise instanceof Promise, 'createInstance returns with a Promise');
 
         instancePromise.then(instance => {
+            console.log(instance.config);
```
Comment (yukipastelcat) [inline test/qtiCommonRenderer/interactions/pci/ims/test.js:62]: Is this `console.log` needed?

**Review by yukipastelcat — APPROVED** [review]
Everything looks good, just check the comment on `console.log` call

**Review Checklist**
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

File: test/qtiCommonRenderer/interactions/pci/ims/test.js (hunk 1 above)
Comment (shaveko) [reply test/qtiCommonRenderer/interactions/pci/ims/test.js]: Thanks, removed that

**Pull request merged**